chart/kubeapps/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubeapps
-version: 4.0.4
+version: 4.0.5
```

Comment:
Maybe land this after @antgamdia 's PR with the ClusterRole, so they're both included in the chart update (once you've reviewed his).

Reply:
well, @antgamdia changes cannot be used until the new endpoint (`/refresh`) is available so the change in the chart is not going to be usable yet.